.github/workflows/continuous-integration-workflow.yml

```diff
@@ -3,9 +3,9 @@ on:
   push:
     branches:
       - master
-  # schedule:
-    # At 15:00 UTC / 7:00 PST on every day-of-week from Monday through Thursday.
-    # - cron: '0 15 * * 1-4'
+  schedule:
+    # At 15:00 UTC / 7:00 PST on every day-of-week from Monday through Friday.
```

Comment:
Deploy on Fridays? 😲

Reply:
Indeed :)

See the comment up above. My concern is a writer might assume that scheduled builds happen on friday (since they happen on all the other weekdays) and schedules something to go out then, only to be surprised when it doesn't.

Reply:
_But_ this happens first thing in the morning on Fridays so I think there's less of a concern of folks having to work weekends. I'm more concerned about deploying at 4pm PST on a Friday.